README.md

```diff
@@ -1664,6 +1664,7 @@ API | Description | Auth | HTTPS | CORS |
 | [Transport for Vancouver, Canada](https://developer.translink.ca/) | TransLink | `OAuth` | Yes | Unknown |
 | [Transport for Washington, US](https://developer.wmata.com/) | Washington Metro transport API | `OAuth` | Yes | Unknown |
 | [Tripadvisor](https://developer-tripadvisor.com/home/) | Rating content for a hotel, restaurant, attraction or destination | `apiKey` | Yes | Unknown |
+| [transport.rest](https://transport.rest) | Community maintained, developer-friendly public transport API | No | Yes | Yes |
```

Comment:
You may want to add that it currently focuses on Germany, although that might change in the future.

Reply:
Thank you for the suggestion!
I currently have an interest in Polish public transport, and I'm actively working on [poland-public-transport-api](https://github.com/konhi/poland-public-transport-api/), so I think that change isn't required!

Reply:
Do you know @MKuranowski and their [list of Polish GTFS feeds](https://mkuran.pl/gtfs/)? They seem very involved with everything related to open public transport data in Poland!

Reply:
Wow, I did not know! That's a big help, thanks so much. I'm really, really impressed of your knowledge about all things related to public transport.